src/components/app-content-switcher/AppContentSwitcher.jsx

```diff
@@ -0,0 +1,49 @@
+import { Typography } from '@mui/material'
+import Tooltip from '@mui/material/Tooltip'
+import Switch from '@mui/material/Switch'
+import Stack from '@mui/material/Stack'
+
+const AppContentSwitcher = ({ isStudent, handleChange, tooltipRight, tooltipLeft, spacing, leftText, rightText, colorActive, colorInActive, typographyVariant, styles }) => {
+  const defaultStyles = {
+    display: 'flex',
+    flexDirection: 'row'
+  }
```

Comment:
Move this to styles file, because you use defaultStyles only once in this file

Reply:
i made this because this switch has to be "horisontal" and it uses this default styles for being that by default. and for not to make separate file and just implment them inside the component. maybe you mean to make this style file just in folser of this component ?

Reply:
Yes. Of Course, create styles file for this component in the same folder and pass it there.

Reply:
refactored